Make it clear that unstable options are only available on a nightly toolchain (#5350)
rust-lang/rustfmt: Configurations.md

```

Each configuration option is either stable or unstable.

> Stable options can always be used, while unstable ones are only available on a nightly toolchain, and opt-in.

Reviewer: Thanks for the PR! I think the proposed rewording makes it clearer that unstable options can only be used on nightly. I'd suggest a slight tweak.

```suggestion
Stable options can always be used, while unstable options are only available on a nightly toolchain and must be opted into.
```
